website/blog/2020-05-06-jest-26.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Jest 26: Tick Tock
+author: Christoph Nakazawa
+authorURL: http://twitter.com/cpojer
+authorFBID: 100000023028168
+---
+
+# Jest 26: Tick Tock
+
+When we started rebuilding Jest five years ago our goal was to provide a batteries-included zero-configuration test runner that is approachable for beginners, extensible for almost all testing use cases and scalable to large projects. One of the most important releases was [Jest 15](https://jestjs.io/blog/2016/09/01/jest-15) which tied all the bits and pieces together and provided great defaults that allowed people to install and run Jest often without any setup. However, this approach has a big downside as Jest installs a lot of dependencies into your projects that you may not need.
+
+We are now beginning to address this shortcoming and are working on reducing Jest’s install size while keeping it approachable and extensible. We have made the following breaking changes in Jest 26:
+
+<!--truncate-->
+
+* `[expect, jest-mock, pretty-format]` Remove `build-es5` from all packages ([#9945](https://github.com/facebook/jest/pull/9945))
+* `[jest-config, jest-resolve]` Remove support for `browser` field ([#9943](https://github.com/facebook/jest/pull/9943))
+* TypeScript definitions requires a minimum of TypeScript v3.8 ([#9823](https://github.com/facebook/jest/pull/9823))
+
+With the above changes Jest 26 is now 4 MiB smaller than Jest 25.5.4 (53 → 49 MiB). Please keep in mind that many dependencies like Babel are likely already part of your project. Jest's own size was reduced from 4.3 MiB to 3.1 MiB.
+
+While this is a good start, it isn’t enough to make Jest meaningfully leaner. We are planning on gradually reducing Jest's size by about 70%. Most of the changes to reduce Jest’s default setup will be small breaking changes and we want to bring the community with us as much as possible. Therefore we will make the following changes in Jest 27 and Jest 28 using a [“Tick-Tock" release process](https://en.wikipedia.org/wiki/Tick%E2%80%93tock_model):
```

Comment:
I think 70% is reasonable if we remove Babel, Jasmine and JSDOM from the default install. Removing all of them likely has a bigger effect than removing them one by one as some transitive deps are probably shared.